core/templates/issue_pages.html

```diff
@@ -76,7 +76,7 @@
                           Issue Notes:
                           {{page_info.issue.noteAboutReproduction.text}}.
                         {% endif %}
-                        {% url 'openoni_issue_pages' title.lccn page_info.issue.date_issued issue.edition as url %}
+                        {% url 'openoni_issue_pages' title.lccn page_info.issue.date_issued 1 as url %}
```

Comment:
I think you were on the right track here. From looking at the code for the URL a few lines above here, I think the appropriate fix is to replace `1` with `page_info.issue.edition`.